tock/tock/templates/_navigation.html

```diff
@@ -1,7 +1,7 @@
 <nav role="navigation" class="usa-nav">
   <div class="usa-nav-inner">
     <button class="usa-nav-close">
-      <img src="/static/img/uswds/close.svg" alt="close">
+      <img src="/tock/static/img/uswds/close.svg" alt="close">
```

Comment:
Wouldn't you just want to use {{ STATIC_URL }} or something like that?

Reply:
Thanks. Updated it in a0b98fd